streetcrm/views.py

```diff
@@ -387,6 +391,26 @@ def process_row(self, row):
                 row[field] = value.strip()
         return row
 
+    def import_data(self, rows):
+        # Create list of objects to create in bulk
+        objects_to_create = []
+        for row in rows:
+            # Use field queries (with default of the key text) to check for
+            # record existence
+            object_query = self.model.objects.filter(
+                **{self.field_queries.get(key, key): value for key, value in row.items()}
+            )
+            # If no objects are returned, add the object to the list to create
+            if not object_query:
+                # objects_to_create.append(self.model(**row))
+                objects_to_create.append(row)
+
+        # Create all new objects
+        # FIXME: Would bulk create, but Django 1.8 doesn't return PKs
```

Comment:
Does this mean we should be upgrading to a more recent Django?  I think there were reasons we didn't (probably some breaking changes?), but at some point we'll need to.  It looks like we're 3 minor versions behind now.

Reply:
I think it'd be a good idea to upgrade, and 1.11 is the most recent version and LTS, but I also don't know if there are any deprecated features we're using in the app